src/components/NewCourse.js

```diff
@@ -104,6 +110,48 @@ class NewCourse extends React.Component {
             </FormText>
           </Col>
         </FormGroup>
+        <FormGroup row>
+          <Label for="name" sm={3}>
+            Unlisted Course
+          </Label>
+          <Form>
```

Comment:
This shouldn't need another `Form` wrapping it - in fact, I think nesting a `<form>` within a `<form>` is invalid HTML5.